### PR TITLE
Feature/convert get-data component

### PR DESCRIPTION
**Pull request by mr-nick17**
### What

- Updated to latest `dp-renderer` and latest `dp-design-system` commit hash
- Converted 'get-data' component from links to radio buttons 
- Added form and validation
- Updated tests
- Removed redundant code 

### How to review

- You need cantabular pages setup locally, follow [these instructions](https://github.com/ONSdigital/dp-compose/tree/main/cantabular-import#readme)
- Compare 'get-data' component to figma design/images below 
- It's unlikely that the backend changes will be in place to "see" getting data for real, insert this code into `handlers.go` at line `191` to mock it 
```go
downloads := map[string]dataset.Download{
	"XLS": {
		Size: "78600",
		URL:  "https://download.ons.gov.uk/downloads/datasets/cpih01/editions/time-series/versions/15.xlsx",
	},
	"CSV": {
		Size: "554700",
		URL:  "https://download.ons.gov.uk/downloads/datasets/cpih01/editions/time-series/versions/15.csv",
	},
	"CSVW": {
		Size: "1900",
		URL:  "https://download.ons.gov.uk/downloads/datasets/cpih01/editions/time-series/versions/15.csv-metadata.json",
	},
	"JSON": {
		Size: "10900",
		URL:  "https://download.ons.gov.uk/downloads/datasets/cpih01/editions/time-series/versions/15.csv-metadata.json",
	},
	"TXT": {
		Size: "2600",
		URL:  "https://www.ons.gov.uk/datasets/cpih01/editions/time-series/versions/15/metadata.txt",
	},
}

version.Downloads = downloads
```
- Check downloads work
- Check validation works
- Sense check
- Tests pass 

#### On landing (previously)
image

#### On landing (new)
image

#### Form validation error
image

image

### Who can review

Frontend dev

### CODE REVIEW

File: assets/templates/census-landing.tmpl [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <div class="ons-page__container ons-container">
     <div class="ons-grid ons-js-toc-container ons-u-ml-no">
         {{ template "partials/breadcrumb" . }}
+        {{ if .Error.Title }}
```
Comment (JoshSinghONS) [inline assets/templates/census-landing.tmpl:4]: Might be wrong but is .Error right? Is it not just Error?

File: assets/templates/partials/census/get-data.tmpl [hunk 2]
```diff
@@ -1,35 +1,50 @@
...
+    {{ if .Error.Title }}
+    <div class="ons-panel ons-panel--error ons-panel--no-title" id="select-format-error">
+        <span class="ons-u-vh">Error: </span>
+         <div class="ons-panel__body">
+    <p class="ons-panel__error">
+      <strong>{{ localise "GetDataValidationError" .Language 1 }}</strong>
+    </p>
+    {{ end }}
+    <form method="get">
+        <input type="hidden" name="f" value="get-data">
```
Comment (JoshSinghONS) [inline assets/templates/partials/census/get-data.tmpl:13]: What's this hidden input for/do?

File: handlers/handlers.go [hunk 3]
```diff
@@ -158,6 +158,9 @@ func censusLanding(ctx context.Context, w http.ResponseWriter, req *http.Request
 	var initialVersion dataset.Version
 	var initialVersionReleaseDate string
 	var err error
+	var form = req.URL.Query().Get("f")
```
Comment (JoshSinghONS) [inline handlers/handlers.go:161]: I assume the hidden input is linked to this but hmmmm

**Review by JoshSinghONS — APPROVED** [review]

File: assets/templates/census-landing.tmpl (hunk 1 above)
Comment (mr-nick17) [reply assets/templates/census-landing.tmpl:4]: It's a prop on the model

File: assets/templates/partials/census/get-data.tmpl (hunk 2 above)
Comment (mr-nick17) [reply assets/templates/partials/census/get-data.tmpl:13]: To identify which form is doing the GET

File: handlers/handlers.go (hunk 3 above)
Comment (mr-nick17) [reply handlers/handlers.go:161]: Yeah, not creatively named though. Just checking that it's the right form

**Pull request merged**